src/components/icon/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
I don't think we should use the `index.js` pattern when the component is not what's meant to be imported from outside the directory.

In cases like this one, it seems like `index.js` should look more like this:

```
import Add from './add';
import Add from './close';

export { Add, Close };
```
(note that there's no component in it) with a directory layout like this:
```
.
..
index.js
icon-container.js
add.js
close.js
```

so that from elsewhere, I could have:

```
import { Add } from '../components/icons';
```

Reply:
I like that suggestion about `index.js` and the icon folder structure.
Thanks for the visual feedback, 

1. I left this a bit unstyled because it was related to the changes I wanted to make- with the switch between "edit" and "view" which no longer seems useful to keep the same way.
2. If there's no relationships the "select" probably should not appear at all, but with a "no relationships" message similar to the "no results".
3. I did disable the button on purpose if nothing was selected but hiding it may be better.
4. No, it was a side-effect :) But I planned to do some big changes on rows in a later PR so I didn't fix it. 
5. The whole "view / edit" workflow needs improvement, I agree it doesn't make sense to have to click _update_ and _done_. I thought about making each filter have an "inline" edit where clicking the button (not the **x**) would open the form. The empty value is a special case where it does not get added to the URL. I put in markup for an error message when it is blank, but it should just be avoided entirely with better UX!
6. The yellow is hard to see, I like that suggestion to use a different icon.
7. thanks!
8. Do you think this whole section should be collapsible? it does need some padding as-is but I was trying to keep it as short as possible so as to not squeeze results down too much.